src/ControlSystem/ControlErrors/Size/DeltaR.cpp

```diff
@@ -96,6 +103,14 @@ std::string DeltaR::update(const gsl::not_null<Info*> info,
        << std::abs(update_args.control_error_delta_r) << " > threshold "
        << delta_r_control_error_threshold << ". Staying in DeltaR.\n";
     ss << " Suggested timescale = " << info->suggested_time_scale;
+  } else if (update_args.average_radial_distance >
```

Comment:
If you make the change about `Options::Auto`, then you need an additional check here `else if (update_args.state_5_stuff.has_value() and ...)` (and in `Initial`)